### ISSUE TITLE
Inserting  via UI data explorer causes error

### ISSUE THREAD

**Issue by davidfries** [edgedb/edgedb-ui#95]
EdgeDB CLI 2.2.6+b869667

When trying to edit any field in the UI when doing an insert with the data explorer, a rendering error is thrown.

```
TypeError: S is null

Traceback:
BQ<@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:1461195
hi/s/<@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:494755
ui/<@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:493873
lt@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:451974
t.track@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:456641
ui@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:493849
s@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:494733
sa@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:289527
Wa@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:299094
Ua@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:298426
Gs@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:345122
Iu@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:329070
ju@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:328998
Eu@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:328859
bu@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:325825
3839/Yi/<@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:275319
4034/t.unstable_runWithPriority@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:368341
Qi@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:275096
Yi@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:275266
Gi@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:275199
Ne@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:346366
Kt@http://localhost:10700/ui/static/js/main.425cd0c5.js:2:253819
```

**Comment by 1st1** [comment]
Would you be able to share your schema and/or make a video of how this happens?

**Comment by davidfries** [comment]
Hi Yury,
I can share a video of it happening. I had my colleague re-produce it, as well. It doesn't matter what object you're trying to insert, it happens for all. Double-clicking to insert data into a field causes the render error.

![edgedbui](https://user-images.githubusercontent.com/12002376/206785039-c42664a6-1b3c-4988-93dc-7f4dc2f69a84.gif)

**Comment by jaclarke** [comment]
Ah, looks like I broke inserts with this fix: https://github.com/edgedb/edgedb-ui/pull/84. Should be fixed by this PR though: #96.

**Comment by AnsonHwang86** [comment]
Hello, When this fixed will publiced? There is a bug in edgedb CLI either, we can specify the version of edgedb. It will always use v2.8 instead of version we typed. So we can't use v2.7 now, please publish this fixes asap in v2.8. thanks.

**Comment by jaclarke** [comment]
There should be a new release containing the fix this week.
